Flaky ecsobserver tests TestExtensionStartStop on windows

```
{"level":"error","msg":"ECSDiscovery stopped by error","error":"code ClusterNotFoundException message Want cluster not used but the mock cluster is different than default config"}
--- FAIL: TestExtensionStartStop (0.03s)
    --- FAIL: TestExtensionStartStop/critical_error (0.02s)
        extension_test.go:110: 
            	Error Trace:	extension_test.go:110
            	Error:      	An error is expected but got nil.
            	Test:       	TestExtensionStartStop/critical_error
```

https://app.circleci.com/pipelines/github/open-telemetry/opentelemetry-collector-contrib/16115/workflows/5bd64a83-ae1d-4309-8a7b-5b534b7c3134/jobs/140322

--- a/extension/observer/ecsobserver/extension_test.go
+++ b/extension/observer/ecsobserver/extension_test.go
@@ -75,7 +75,7 @@ func (h *inspectErrorHost) getError() error {
 // In that case sd itself does not use timer and relies on caller to trigger List.
 func TestExtensionStartStop(t *testing.T) {
 	settings := component.ExtensionCreateSettings{Logger: zap.NewExample()}
-	refreshInterval := time.Millisecond
+	refreshInterval := 100 * time.Millisecond
 	waitDuration := 2 * refreshInterval
 
 	createTestExt := func(c *ecsmock.Cluster, output string) component.Extension {

